City-of-Helsinki/employment-services-ui
# THF-482: Add multi filtering to events

Created possibility to Filter events with multiple values.

How to test:

- Filter events with different option ->  you should get all events which have all the tag you filtered. 
- Choose all filters -> When filter value is 0 you should get info "no search result"
- Toggle the filter tag -> When you click event tag button first time the value goes to the filter arrays. When you click the same button second time the tag will be removed from the filter array.
